python/observability-online-eval/pydantic-ai/simple_agent.ipynb

```
@@ -0,0 +1,199 @@
+     ]
+    }
+   ],
+   "source": [
+    "# Instrument Pydantic AI once at the top\n",
+    "print(\"Initializing Maxim instrumentation for Pydantic AI...\")\n",
+    "instrument_pydantic_ai(maxim_logger, debug=True)\n",
+    "print(\"Instrumentation complete!\")\n",
+    "\n",
+    "# Import Pydantic AI components\n",
+    "try:\n",
+    "    from pydantic_ai import Agent, RunContext\n",
```

Ruff (0.12.2)

`await` statement outside of a function

(F704)